paper.md

```diff
@@ -0,0 +1,88 @@
+
+![Figure 1. A CT image correctly aligned to the MR image. The darker areas in the MR are the skull whereas the lighter, outermost regions are subcutaneous fat. The CT is shown with a red scale, and red areas shown in the slice are the skull which has high intensity in the CT.](figures/Figure_1.png)
+
+Next, electrode contact locations are selected in merged CT-MR coordinates using a graphical user interface (GUI) implemented in PyQt. In the GUI, users click or scroll through slices to find the location of contacts. When the user navigates to the location of a channel, the channel name can be selected from a menu listing the names extracted from the recording file. Then, the user can mark the channel as associated with that location. The channel is then rendered in the 3D view as well as colored on the slice plots.
+
+![Figure 2. A screenshot of the GUI during a sEEG electrode location. For sEEG, the 3D brain view is typically similar to the surgical plans, making it easier to determine which electrode is which. With only 2D slice plots, locations and directions must be pieced together which is substantially more difficult.](figures/Figure_2.png)
+
+![Figure 3. A screenshot of the GUI during an ECoG electrode location. For ECoG, the 3D brain view makes it easier to select each contact in a large grid. Without the 3D view, it is difficult to continue in one row without jumping to another row incorrectly when viewing in 2D. These mistakes make it much slower to identify contacts.](figures/Figure_3.png)
+
+Located electrode contacts in merged CT-MR coordinates for an individual subject can be morphed to a template brain for group analysis. This is done using a symmetric diffeomorphic registration [SDR; @GaryfallidisEtAl2014:2014]. This non-linear mapping tends to be more accurate than a linear transformation such as the Talairach transform [@DaleEtAl1999:1999] as shown in Figure 4.
+
+![Figure 4. a) Example sEEG electrodes in a 3D rendering of individual subject anatomy. b) The electrodes mapped to the Freesurfer ``fsaverage`` brain using the SDR morph. c) The electrodes mapped to the Freesurfer ``fsaverage`` brain using the Talairach transform. The linear transformation in ``c`` is less accurate than the SDR transform in ``b`` which can be seen by the electrode in the temporal pole floating outside the pial surface in ``c``.](figures/Figure_4.png)
+
+For ECoG electrodes, "brain shift", caused by changes in pressure during the craniotomy, can also be accounted for using MNE-Python. This shift causes grid and strip electrodes to be deeper in the post-operative CT than they were in the preoperative MR. This makes it so that the electrodes are inside the pial surface in the preoperative MRI. In MNE-Python, this is compensated for by projecting the grid to the leptomeningeal surface.
+
+![Figure 5. a) A 3D rendering of an ECoG grid without correction for "brain shift". b) The same 3D rendering with the "brain shift" correction.](figures/Figure_5.png)
```

Comment:
For a, I think it would be worth adding "Note that in some cases electrodes are rendered below the pial surface". I know it is somewhat obvious, but still think it is worth including.